### PR TITLE
Add link to Build's sha

### LINKED ISSUE #708: Link the commit shas to their code hosting page
Parent issue #682

### PR DESCRIPTION
**Pull request by minuscorp**
Fixes #708 

captura de pantalla 2018-04-24 a las 20 56 35

### CODE REVIEW

File: web/app/common/test_helpers/mock_build_data.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export const mockBuildSummaryResponse_failure: BuildSummaryResponse = {
   status: 'failure',
   duration: 1234,
   sha: 'cjsh4',
+  link_to_sha: 'https://github.com/fastlane/ci/commit/1015c506762b1396a5d63fff6fe0f1de43c8de80',
```
[the remark this answers is not in the document]
Comment (KrauseFx) [reply web/app/common/test_helpers/mock_build_data.ts]: Yesss 🚀

**Comment by minuscorp** [comment]
The backend is actually building that link by now. Was added a few PR's ago, as it was related to the model and its code hosting provider.

File: web/app/project/project.component.html [hunk 2]
```diff
@@ -27,8 +27,11 @@
         <!-- Lane Column -->
         <ng-container matColumnDef="sha">
           <mat-header-cell *matHeaderCellDef>Commit Sha</mat-header-cell>
-          <!-- TODO: link this to the GitHub commit -->
-          <mat-cell *matCellDef="let build">{{build.shortSha}}</mat-cell>
+          <mat-cell *matCellDef="let build">
+            <a href="{{build.linkToSha}}">
```
Comment (nakhbari) [inline web/app/project/project.component.html]: `[href]="build.linkToSha"`

Comment (nakhbari) [reply web/app/project/project.component.html]: actually i think it's `[attr.href]`

Comment (nakhbari) [reply web/app/project/project.component.html]: Do you think we should open this in a new tab?

Comment (minuscorp) [reply web/app/project/project.component.html]: let's be greedy, new tab incoming 😃

**Pull request merged**